owl/EASE-OBJ.owl

```diff
@@ -449,6 +651,15 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Shape -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Shape">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Region"/>
```

Comment:
Given that Cylinder etc. are subclasses of Shape, it is certainly appropriate to have Shape be a subclass of Region. What is the Quality that Shape is the Region for?

Reply:
Appearance for example. But in the case of URDF the visual or the collision model